MapleServer2/PacketHandlers/Game/GroupChatHandler.cs

```diff
@@ -75,13 +75,9 @@ private static void HandleInvite(GameSession session, PacketReader packet)
             }
 
             int groupChatCheck = 0;
-            for (int i = 0; i < 3; i++)
-            {
-                if (other.GroupChatId[i] != 0)
-                {
-                    groupChatCheck++;
-                }
-            }
+            int count = other.GroupChatId.Count(x => x != 0);
+            
+            if (count == 3)
```

Comment:
🤔

Reply:
😕 I just merged it with this fix that was suggested, but yeah it's throwing an error. I'm not too familiar with this new approach so I don't know what would be the fix for it or if there's a better fix than what I originally had.